README.md

```diff
@@ -2866,6 +2874,126 @@ keyword 'default'.
 Virtual ip configuration. Valid values are an array of Strings or 'default'.
 
 --
+### Type: cisco_object_group
+
+Manages configuration of an ObjectGroup instance. This has no properties and it is the parent of ObjectGroupEntry.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I2(2e)       | 1.8.0                  |
+| N3k      | 7.0(3)I2(2e)       | 1.8.0                  |
+| N5k      | not applicable     | not applicable         |
+| N6k      | not applicable     | not applicable         |
+| N7k      | 7.3(0)D1(1)        | 1.8.0                  |
+| N9k-F    | 7.0(3)F1(1)        | 1.8.0                  |
+
+#### Parameters
+
+| Example Parameter Usage
+|:--
+| `cisco_object_group { '<afi> <type> <grp_name>':`
+| `cisco_object_group { 'ipv4 address myog_v4_addr':`
+
+##### `afi`
+Address Family Identifier (AFI). Required. Valid values are ipv4 and ipv6.
+
+##### `type`
+Type of the object_group instance. Required. Valid values are address and port.
+
+##### `grp_name`
+Name of the object_group instance. Required. Valid values are type String.
+
+##### Properties
+
+##### `ensure`
+Determines whether the config should be present or not on the device. Valid values are 'present' and 'absent'.
+
+--
+### Type: cisco_object_group_entry
+
+Manages configuration of an ObjectGroupEntry instance.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I2(2e)       | 1.8.0                  |
+| N3k      | 7.0(3)I2(2e)       | 1.8.0                  |
+| N5k      | not applicable     | not applicable         |
+| N6k      | not applicable     | not applicable         |
+| N7k      | 7.3(0)D1(1)        | 1.8.0                  |
+| N9k-F    | 7.0(3)F1(1)        | 1.8.0                  |
+
+#### Example Usage
+
+```puppet
+cisco_object_group_entry { 'ipv4 address myoge_v4_addr 10':
+  ensure              => 'present',
+  address             => '10.10.10.1/24',
+}
+
+cisco_object_group_entry { 'ipv4 port myoge_v4_port 20':
+  ensure              => 'present',
+  port                => 'neq 40',
+}
+
+cisco_object_group_entry { 'ipv6 address myoge_v6_addr 30':
+  ensure              => 'present',
+  address             => '2000::1/64',
+}
+```
+
+#### Parameters
+
+| Example Parameter Usage

```

Comment:
You already have the Example usage above.  Not sure this Example Parameter Usage section adds any value to that.